IdentityCore/src/workplacejoin/ios/MSIDWorkPlaceJoinUtil.m

```diff
@@ -123,7 +123,7 @@ + (MSIDRegistrationInformation *)getRegistrationInformation:(id<MSIDRequestConte
     identity = [self copyWPJIdentity:context sharedAccessGroup:sharedAccessGroup certificateIssuer:&certificateIssuer privateKeyDict:&keyDict];
     if (!identity || CFGetTypeID(identity) != SecIdentityGetTypeID())
     {
-        MSID_LOG_WITH_CTX(MSIDLogLevelInfo, context, @"Failed to retrieve WPJ identity. Identity %@, typeIdMismatch %@", @(!identity), @(CFGetTypeID(identity) != SecIdentityGetTypeID()));
+        MSID_LOG_WITH_CTX(MSIDLogLevelInfo, context, @"Failed to retrieve WPJ identity. Identity %@", @(!identity));
```

Comment:
Maybe identity == nil will be clearer?

Reply:
Will update in this PR: https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/1184